fdbt-dev/sql/passengerTypes.sql

```diff
@@ -0,0 +1,14 @@
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+LOCK TABLES `passengerType` WRITE;
+
+TRUNCATE TABLE `passengerType`;
+
+INSERT INTO `passengerType` (name,contents,isGroup,nocCode) 
+VALUES
+('Small People', '{"id":"","passengerType":"child","ageRangeMin":"","ageRangeMax":"18"}', 0, 'BLAC'),
```

Comment:
Why are we leaving ID from the Insert and from the contents?

Reply:
id we can't insert as this is auto increment and the contents contain the data in this format ...

Reply:
i created the passenger type from the UI and use that one ... I have to check if we delete the id from single passenger type then it should not break